.github/workflows/rust.yml

```diff
@@ -102,10 +102,10 @@ jobs:
             emulator-arch: x86_64
             # Note that x86_64 image is only available for API 21+. See
             # https://github.com/ReactiveCircus/android-emulator-runner?tab=readme-ov-file#configurations.
-            api-level: 21
+            api-level: 26
           - target: i686-linux-android
             emulator-arch: x86
-            api-level: 19
+            api-level: 26
```

Comment:
The `ecn_*` unit tests are failing with any Android API version < v26 with:

```
2024-08-22T14:41:30.636494Z  WARN quinn_udp: sendmsg error: Os { code: 22, kind: InvalidInput, message: "Invalid argument" }, Transmit: { destination: 127.0.0.1:50196, src_ip: None, enc: Some(Ect0), len: 5, segment_size: None }
```

I will open a separate pull request discussing this failure.

I suggest we merge here already. Some Android tests (i.e. >=26) are better than none.

Reply:
> I will open a separate pull request discussing this failure.

For the record, the separate pull request: https://github.com/quinn-rs/quinn/pull/1975